### PR TITLE
dev-python/markuppy: fix QA warning: description-file

### PR DESCRIPTION
**Pull request by scardracs**
Drop python3.7 and add support for 3.10
Fix license

Closes: https://bugs.gentoo.org/798060
PR upstream: https://github.com/tylerbakke/MarkupPy/pull/4

Package-Manager: Portage-3.0.20, Repoman-3.0.3
Signed-off-by: Marco Scardovi <marco@scardovi.com>

### CODE REVIEW

File: dev-python/markuppy/markuppy-1.14-r1.ebuild [hunk 1]
```diff
@@ -17,8 +17,13 @@ SRC_URI="mirror://pypi/${MY_PN:0:1}/${MY_PN}/${MY_P}.tar.gz"
...
-LICENSE="BSD-2"
+LICENSE="MIT"
 KEYWORDS="~amd64"
 
 # MarkupPy does not have any test suite
 RESTRICT="test"
+
+src_prepare() {
+	eapply_user
+	sed -i -e 's/description-file/description_file/' setup.cfg || die "Failed to fix setup.cfg"
```
Comment (mgorny) [inline dev-python/markuppy/markuppy-1.14-r1.ebuild:28]: This needs to be fixed upstream, not patched in Gentoo.

Comment (scardracs) [reply dev-python/markuppy/markuppy-1.14-r1.ebuild:28]: I know, in fact I've opened a PR upstream but seems dead since 2017

**Comment by scardracs** [comment]
@mgorny is it ok if I fork it and fix myself?

**Comment by mgorny** [comment]
Are you actually going to maintain that fork going forward? I dare say changing setup.cfg key is not a good reason itself for forking.

**Comment by scardracs** [comment]
> Are you actually going to maintain that fork going forward? I dare say changing setup.cfg key is not a good reason itself for forking.

mmh, you're right. Unfortunately upstream seems dead. I will send a mail to maintainer and see if he answer to me

**Comment by scardracs** [comment]
@mgorny the markuppy setup.cfg is now fixed upstream but the version in github completely miss the MarkupPy directory (pypi has it). I've asked upstream if he can release a new version on pypi too

**Comment by scardracs** [comment]
Close as the package is not required anymore (it was a netbox requirement)

**Pull request closed without merging**